### PR TITLE
fix block size limit issue

### PR DESCRIPTION
**Pull request by bird512**

### CODE REVIEW

File: core/transaction_pool.go [hunk 1]
```diff
@@ -244,9 +244,12 @@ func (pool *txPool) packTx() []*types.Transaction {
 	txs := make([]*types.Transaction, 0)
 	accuSize := 0
 	pool.bonPool.forEach(func(tx *types.Transaction) bool {
-		txs = append(txs, tx)
 		accuSize += tx.Size()
-		return accuSize < txAccumulateSizeMaxPerBlock
+		if accuSize < txAccumulateSizeMaxPerBlock{
```
Comment (sesamegu) [inline core/transaction_pool.go]: <改为 <=

File: core/transaction_pool.go [hunk 2]
```diff
@@ -255,9 +258,12 @@ func (pool *txPool) packTx() []*types.Transaction {
 			if tx.GasPrice.Cmp(pool.gasPriceLowerBound.Value()) < 0 {
 				return true
 			}
-			txs = append(txs, tx)
 			accuSize = accuSize + tx.Size()
-			return accuSize < txAccumulateSizeMaxPerBlock
+			if accuSize < txAccumulateSizeMaxPerBlock {
```
Comment (sesamegu) [inline core/transaction_pool.go]: <改为 <=

**Review by sesamegu — CHANGES_REQUESTED** [review]

File: core/transaction_pool.go (hunk 2 above)
Comment (bird512) [reply core/transaction_pool.go]: resolved

File: core/transaction_pool.go (hunk 1 above)
Comment (bird512) [reply core/transaction_pool.go]: resolved

**Review by liuhaox1 — APPROVED** [review]

**Review by sesamegu — APPROVED** [review]

**Pull request merged**